ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml

```diff
@@ -234,13 +258,27 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-techpreview-parallel
+  cron: 0 2,10,18 * * *
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      TEST_SUITE: openshift/conformance/parallel
+    workflow: openshift-e2e-openstack-techpreview
```

Comment:
I thought we had a job already for that:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/openstack/ccm/openshift-e2e-openstack-ccm-workflow.yaml

@mdbooth @mandre

Reply:
That one is specific for CCM, this PR is adding generic `TechPreviewNoUpgrade` tests to prove out TP features in general and in the future, CCM just happens to be a subset of that for now